### ISSUE TITLE
Colored sqaures in tables alt text needs to be changed to "decorative" or ""

### ISSUE THREAD

**Issue by eelzi-ONRR** [DOI-ONRR/nrrd#2729]
The squares don't add meaning  - everything the user needs is in the table data. Only meaningful images should have descriptive alt text

![image.png](https://images.zenhubusercontent.com/63482c003bae1bf4351c6f26/6ca84a07-48bd-447e-93ae-3a9d0c20ba63)

**Comment by eelzi-ONRR** [comment]
I researched this and svgs should not have alt text (they should only have titles) and so they cannot be marked as decorative. Instead we need to mark them as hide them from screen readers:

aria-hidden="true"

@greensomes would that go in this tag since it applies to all of these squares on the site? [Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMEI4QVE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--0b82875141399a8574483596d1999c6e910e1d97/image.png)

**Comment by greensomes** [comment]
Hey, @eelzi-ONRR. In the past, we have set the title property for SVGs. For this issue, is it enough to update the title property, or do we need to set the aria-hidden attribute? If we're setting the aria-hidden attribute, do we need to go through all of the other SVGs and update those as well?

**Comment by eelzi-ONRR** [comment]
@greensomes The problem is that these are decorative. Giving meaning to an image (whatever format - in this case SVG) is against 508. While we can mark images files with alt text of "" to indicate decorative and subsequently the screen reader will skip it and not waster the users time. Screen readers won't read an svg title in the same way.  That's why best practice is to use an aria label to hide it from screen readers all together.

We don't need to do this for all SVGs - most of our SVGs aren't decorative. But all instances of these squares are decorative so they all need to be hidden

**Comment by greensomes** [comment]
Got ya! Are all the instances of these squares documented? I just need to know the URLs of the pages where these are.

**Comment by eelzi-ONRR** [comment]
@greensomes Here's the three pages they are on, along with the tabs/filtered pages and the number of times they occur
- https://revenuedata.doi.gov/ 
Tabs: Revenue, Disbursements, Production.	1 table on each tab

- https://revenuedata.doi.gov/fact-sheet/  
6 tables

- https://revenuedata.doi.gov/explore 
-**Revenue** - Comparison cards (2 tables), Nationwide revenue summary (6 tables), Top Nationwide locations (1 table)
-**Production** - Comparison cards (1 table), Top Nationwide locations (1 table)
-**Disbursements** - Comparison cards (2 tables), Nationwide Disbursements summary (7 tables),  Top Nationwide Recipients (1 table)

**Comment by greensomes** [comment]
Hi @eelzi-ONRR, I added the aria-hidden attribute to each instance of the colored squares you listed above. Here's the [preview link](https://nrrd-preview.app.cloud.gov/sites/2729-colored-squares).

Is it necessary to remove the `title` tag from the `svg` elements as well?

**Comment by eelzi-ONRR** [comment]
@greensomes I'm finding that the squares are hidden on the fact sheet but not the home page or explore. I checked with both ANDI and WAVE.

No need to remove the title tag.
